pages/content/reports/2022/index.md

```diff
@@ -0,0 +1,391 @@
+
+![background property formal syntax before improvement](formal-syntax-background-before.png)
+
+...now looks like this:
+
+![background property formal syntax after improvement](formal-syntax-background-after.png)
+
+These two changes - more accurate content, clearly presented - make it much easier for web developers to work with CSS.
+
+This project greatly benefited from OWD's close connections with W3C. While it serves the users of MDN, it also enables OWD to provide feedback to W3C on how they can make specifications more accessible.
+
+## Write the Docs
+
+In May, OWD hosted a session at the[ Writing Day](https://www.writethedocs.org/conf/portland/2022/writing-day/) during the[ Portland 2022 edition of the Write the Docs conference](https://www.writethedocs.org/conf/portland/2022/). Write the Docs is a conference for people involved with all aspects of documentation, including writers, developers, editors, information architects and more.
+
+For the Writing Day OWD prepared a project to remove `var` from code samples in the JavaScript documentation on MDN, replacing it with `const` or `let` as appropriate. We were joined by about 20 volunteers, most of whom had never made a GitHub pull request before. With our help, volunteers removed `var` from 254 pages in the JavaScript reference.
+
+Eventually, this project turned into a much larger project to modernize all MDN’s JavaScript code samples across the whole site.
+
+## Shared Investment is Lasting Investment?
+
+From end-user applications to developer tools to new APIs, developing for the web platform relies on access to accurate, vendor-neutral information and how-to. Thanks to the generous support of our steering committee participants and funding partners, Open Web Docs has:
+
+- Retained key talent, including some of the original architects of MDN, and engaged them on core projects that have
+  - Grown the impact of web documentation by making it more relevant, contextual, and available for everyday developers
+  - Reinforced the collective community management and caretaking of web platform documentation - no single company could begin to recreate these resources, or manage them alone
+  - Supported new contributors, mentoring tomorrow’s top-class tech talent on best practices and content development at web scale
+- Spread collective responsibility and investment in common digital infrastructure among multiple vendors, which
+  - Reduces risk to the project
+  - Increases visibility and “ownership” behaviors among invested parties
+  - Strengthens reliability, longevity, and trustworthiness
+  - Reduces costs
+- Improved breadth and quality of the information and data on
+  - Core reference documentation
+  - Browser Compatibility
+  - Hot-topic technologies for next generation application development such as AR/VR, security, and privacy-protecting technologies
```

Comment:
I think we can leave out AR/VR here given WebXR was a focus of 2021 and not 2022. We are still working on documenting new tech, though.

```suggestion
  - Hot-topic technologies for next generation application development, security, and privacy-protecting technologies
```